presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -220,6 +221,14 @@ public void checkCanCreateView(ConnectorSecurityContext context, SchemaTableName
         }
     }
 
+    @Override
+    public void checkCanRenameView(ConnectorSecurityContext context, SchemaTableName viewName, SchemaTableName newViewName)
+    {
+        if (!checkTablePermission(context, viewName, OWNERSHIP)) {
```

Comment:
I believe we should be checking both the old name and the new name.... it looks like rename table is broken also

Reply:
I have fixed for table also